MvvmCross/Platforms/Android/Views/MvxAndroidViewDispatcher.cs

```diff
@@ -13,22 +14,17 @@ public class MvxAndroidViewDispatcher
         : MvxAndroidMainThreadDispatcher
         , IMvxViewDispatcher
     {
-        private readonly IMvxAndroidViewPresenter _presenter;
-
-        public MvxAndroidViewDispatcher(IMvxAndroidViewPresenter presenter)
```

Comment:
To my eyes the ViewDispatcher object looks easier to override / use if the ViewPresenter is a dependency declared in the constructor.

A property isn't clear enough most of the time, and it can make developers spend quite some time figuring out why their code doesn't work. Is this change necessary?

Reply:
Having constructor with parameters means that you have to override both the implementation and wherever in the framework the instance gets created. Preference is to override the implementation and simply register the new implementation. 
Unfortunately interfaces don't allow us to specify constructors with parameters - if they did, that would be best of both worlds